css/style.css

```diff
@@ -17,3 +17,204 @@ body {
  */
 
 
+ /** Navigation styles */
+ .header-container {
+    background-color: white;
+    border-bottom: 0.5px solid #A8A8A8;
+    height: 80px;
+    width:100%;
+    position: fixed;
+    z-index: 999;
+    top:0px;
+    left:0px;
+}
+
+.header-inner {
+    background-color:white;
+    height:80px;
+    /**width: 1000px */
+    width:80%;
+    margin:auto;
+    display: flex;
+    justify-content: space-between;
+    align-items:center;
+}
+
+ .logo {
+    width: 40px;
+    height: 40px;
+    padding: 10px;
+  }
+
+ .navigation {
+     list-style-type: none; 
+ }
+
+ .navigation li {
+    padding:10px;
+    display:inline-block;
+  }
+
+  .navigation li a {
+    text-decoration: none;
+    color:#A8A8A8;
+    font-weight: 400;
+  }
+
+  .navigation li a:hover {
+    color:black;
+  }
+
+.navigation li .karma-text {
+    text-decoration: none;
+    color:#484848;
+    font-weight:500;
+}
+
+/** Hero styles */
+#hero-container {
```

Comment:
Don't use IDs for CSS! They are not re-usable and can cause style clashing.
Always prefer CSS classes or HTML tags.